### DISCUSSION TITLE
How to start contributing?

### DISCUSSION THREAD

**Discussion by programming-warrior** [RocketChat/Rocket.Chat#30415] [Q&A] [answered]
Hi! I am an CS undergrad student. I want to contribute to rocket.chat project.
Any suggestions, where should I start?
I know typescript, nodejs, javascript.

**Comment by Akshat111111** [comment, marked as answer]
You can start contributing after you have read the documentation throughly . Make sure you know about git/ github basics and how to setup the project in the local machine.

**Comment by Akshat111111** [comment]
You can directly start by good first issues label initially.
